lighthouse-core/test/fraggle-rock/scenarios/stringified-replay-test-pptr.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {promises as fs} from 'fs';
```

Comment:
any reason to use promises for this? typically doesn't ever matter for tests

Reply:
Guess that's how I was feeling, sync is fine